### DISCUSSION TITLE
Q: Use of exchangerate.host

### DISCUSSION THREAD

**Discussion by ekandreas** [worksome/exchange#3] [Ideas]
(sorry, not possible to Q in your repo, gives 404)

Possible to link https://exchangerate.host/ instead that we find both more stabel, faster and realible than Fixer?

**Comment by lukeraymonddowning** [comment]
Looks interesting; seems to have an identical API so might be able to support this with a decorator. @olivernybroe, thoughts?

**Reply by olivernybroe** [reply]
That makes sense. They can prob. share a lot of the same logic then.

**Comment by lukeraymonddowning** [comment]
@ekandreas @olivernybroe I've implemented support for this in the following PR: https://github.com/worksome/exchange/pull/4

**Comment by lukeraymonddowning** [comment]
Released in v0.2.0
